libs/ui-watt/src/lib/components/input/timepicker/watt-timepicker.component.ts

```diff
@@ -121,23 +226,45 @@ export class WattTimepickerComponent extends WattPickerBase {
    */
   protected initRangeInput() {
     // Setup and subscribe for input changes
-    this.rangeInputService.init({
-      startInput: this.maskInput(
-        this.startInput.nativeElement,
-        (this.initialValue as WattRange | null)?.start
-      ),
-      endInput: this.maskInput(
-        this.endInput.nativeElement,
-        (this.initialValue as WattRange | null)?.end
-      ),
+    const startInput = this.maskInput(
+      this.startInput.nativeElement,
+      (this.initialValue as WattRange | null)?.start
+    );
+
+    const endInput = this.maskInput(
+      this.endInput.nativeElement,
+      (this.initialValue as WattRange | null)?.end
+    );
+
+    this.rangeInputService.init({ startInput, endInput });
+
+    // Silence the compiler since `onInputChanges$` is always assigned in `init`
+    const { onInputChanges$ = EMPTY } = this.rangeInputService;
+    const timeRange$ = onInputChanges$.pipe(takeUntil(this.destroy$));
+
+    // Synchronize the slider value with the input fields. Calling `update`
+    // here automatically triggers an emit on the `timeRange$` observable.
+    this.sliderChange$.pipe(takeUntil(this.destroy$)).subscribe((value) => {
+      startInput.maskedInput.update(minutesToTime(value.min));
+      endInput.maskedInput.update(minutesToTime(value.max));
     });
```

Comment:
suggestion: rename variable

```suggestion
    this.sliderChange$.pipe(takeUntil(this.destroy$)).subscribe((sliderValue) => {
      startInput.maskedInput.update(minutesToTime(sliderValue.min));
      endInput.maskedInput.update(minutesToTime(sliderValue.max));
    });
```

Reply:
4940e06fbc9e6384aa62c239060ae69a7a8cb1a2